src/main/java/de/tum/in/www1/artemis/service/connectors/aeolus/AeolusTemplateService.java

```diff
@@ -39,6 +38,8 @@ public class AeolusTemplateService {
 
     private final ResourceLoaderService resourceLoaderService;
 
+    private final HashMap<String, Windfile> templateCache = new HashMap<>();
```

Comment:
Maybe replacing HashMap with ConcurrentHashMap would be a better idea for thread safety

Reply:
you're absolutely right, thanks!